packages/react-import-remote/README.md

```diff
@@ -0,0 +1,135 @@
+# `@shopify/react-import-remote`
+
+[![Build Status](https://travis-ci.org/Shopify/quilt.svg?branch=master)](https://travis-ci.org/Shopify/quilt)
+[![License: MIT](https://img.shields.io/badge/License-MIT-green.svg)](LICENSE.md) [![npm version](https://badge.fury.io/js/%40shopify%2Freact-import-remote.svg)](https://badge.fury.io/js/%40shopify%2Freact-import-remote.svg) [![npm bundle size (minified + gzip)](https://img.shields.io/bundlephobia/minzip/@shopify/react-import-remote.svg)](https://img.shields.io/bundlephobia/minzip/@shopify/react-import-remote.svg)
+
+Provides a utility function and React component to asynchronously load scripts and extract the newly-added global
```

Comment:
I'm not 💯 on this description. Perhaps something like this:

> Asynchronous script loading for React

Also, you'd need to update in the `package.json` too